Mathlib/Combinatorics/Young/YoungDiagram.lean

```diff
@@ -153,7 +153,6 @@ theorem cells_bot : (⊥ : YoungDiagram).cells = ∅ :=
   rfl
 
 -- Porting note: removed `↑`, added `.cells` and changed proof
--- @[simp] -- Porting note (#10618): simp can prove this
 @[norm_cast]
```

Comment:
seems that this was not added again. Is that because `simp` can prove it? If so, that's fine, I'm just making sure it didn't slip through the cracks.

Reply:
Indeed these two ere removed on purpose, see commit `136f54eeeb750f2f064a6ee4d3bcaf9122810ffd`.